### ISSUE TITLE
clipspeak and numpad nav mode functionality inclusion please

### ISSUE THREAD

**Issue by M3198** [nvaccess/nvda#16478]
Yes it is related to a problem, specifically the way certain lack of features may impact certain users of NVDHi,

I hope you are doing well.

I have two suggestions for NVDA.

1. Please include clipspeak functionality, as it's quite important people know when they're selecting, coppying and pasting text. This way, they can easily track what they do.

2. Please incorporate numpad nav mode into NVDA. If you recall, I once had this problem when I purchased my llaptop and you told me to use that addon. Though at the moment it doesn't seem that addon will stop being developed, I think it is still very useful for that functionality to be included especially for those people, like myself, who don't have a dedicated home key on their laptop.

Best regards,

Muhammad.This is the only alternative I have considered.When I got my laptop, my laptop didn't have a dedicated home key. For this reason, the numpad 7 was also the home button. Because of this, I couldn't use the computer properly, such as by jumping to the beginning and end of a webpage, document or the desktop folder itself.A

**Comment by Adriani90** [comment]
1. Clipspeak is not a feature that can be incorporated in NVDA due to following reasons:
* The apparent "copied to clipboard" and "pasted" messages reported by the add-on clipspeak are misleading. Applications do not provide any guarantee that something has been copied. People that just want to hear this message might be confortable that they heard something, but in fact this message is worthless because that doesn't mean you copied something. Note that there are lots of copy actions that can be done but there is no guarantee that it succeeded. For example ctrl+shift+c in MS word to copy only the formating of texts.
* Applications in which a copy action is guaranteed provide the message by themselves, not only for screen readers, but also as a visual indicator that something has been copied.
* Applications which do not provide any message for copied to clipboard natively, they also don't provide any visual indicator, so sighted people face the same problem, they never know exactly if ctrl+c really copied something to the clipboard.

So in summary, I know there are a lot of people who like the clipspeak add-on, but unfortunately its functionality is misleading, sugerating users that something has been copied although no one knows if the copy process was successful.

2. Numpad nav: I filled #16479. Also the automatic turn off for numlock is already covered in #4093.

That being said, I am closing this one.
